lib/rucio/common/policy.py

```diff
@@ -16,20 +16,13 @@
 import logging
 import sys
 
+from functools import wraps
+
 from ConfigParser import NoOptionError, NoSectionError
 from dogpile.cache import make_region
 from dogpile.cache.api import NoValue
 
-import rucio.core.did
-import rucio.core.lock
-import rucio.core.rule
-
-from rucio.db.sqla.constants import DIDType
-from rucio.db.sqla.session import read_session, transactional_session
 from rucio.common.config import config_get
```

Comment:
sort import

Reply:
??? That's sorted. First system import, then other packages

Reply:
hmmm 🔢 
from rucio.common.config import config_get
from rucio.db.sqla.constants import DIDType
from rucio.db.sqla.session import read_session, transactional_session

I think I missed the - removed lines